cnf-certification-test/lifecycle/suite.go

```diff
@@ -372,3 +378,33 @@ func testPodsRecreation(env *provider.TestEnvironment) { //nolint:funlen
 		}
 	}
 }
+
+func testPodRequestsAndLimits(env *provider.TestEnvironment) {
+	var containersMissingRequestsOrLimits []string
+	ginkgo.By("Testing container resource requests and limits")
+
+	// Loop through the containers, looking for containers that are missing requests or limits.
+	// These need to be defined in order to pass.
+	for _, cut := range env.Containers {
+		badContainer := false
+		if len(cut.Data.Resources.Limits) == 0 {
+			tnf.ClaimFilePrintf("Container has been found missing resource limits: %s", cut.String())
+			badContainer = true
+		}
+
+		if len(cut.Data.Resources.Requests) == 0 {
+			tnf.ClaimFilePrintf("Container has been found missing resource requests: %s", cut.String())
+			badContainer = true
+		}
+
+		if badContainer {
+			containersMissingRequestsOrLimits = append(containersMissingRequestsOrLimits, cut.String())
+		}
+	}
+
+	if n := len(containersMissingRequestsOrLimits); n > 0 {
+		errMsg := fmt.Sprintf("Containers found that are missing either resource requests or limits: %d. See logs for more detail.", n)
```

Comment:
If the above applies, the string should reference "requests or limits for cpu/memory".